### DISCUSSION TITLE
the crop version of PadListDataCollate

### DISCUSSION THREAD

**Discussion by jxchen01** [Project-MONAI/MONAI#5909] [General]
Is it possible to do something like "CropListDataCollate"? Similar to PadListDataCollate, but crop all data to the minimum dimensions.

Thanks!

**Comment by Nic-Ma** [comment]
Maybe it would be nice to add a `PadOrCropListDataCollate`, refer to the existing transform:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/croppad/array.py#L1265
Then it can cover the cases: (1) pad to max dims, (2) crop to min dims, (3) resize to specified sizes.
CC @wyli @ericspod @KumoLiu What do you think?

Thanks.

**Reply by ericspod** [reply]
That would generalise the concept I suppose, though I have no specific use for this I can see others wanting it.

**Comment by Nic-Ma** [comment]
@yiheng-wang-nv Could you please help on this new feature request?

Thanks in advance.

**Reply by yiheng-wang-nv** [reply]
Update: I'm starting to work on this feature.
